package.json

```diff
@@ -71,6 +71,7 @@
 	"dependencies": {
 		"colors": "^1.4.0",
 		"console-stamp": "^3.0.0-rc4.2",
+		"digest-fetch": "^1.1.6",
```

Comment:
where is that package used in your changes? cant find any imports of it

Reply:
in `calendarfetcher.js`, the diff is collapsed ...

Reply:
Doh :-)